fix horizontal expansion of textarea with autoexpand={true} (#786)
Doist/reactist: CHANGELOG.md

Reactist follows [semantic versioning](https://semver.org/) and doesn't introduce breaking changes (API-wise) in minor or patch releases. However, the appearance of a component might change in a minor or patch release so keep an eye on redesigns and make sure your app still looks and feels like you expect it.

# v21.0.3

> -   [Fix] `overflow-wrap: anywhere` is added to stop horizontal expansion of `TextArea` with `autoExpand={true}`.

Reviewer: ```suggestion
-   [Fix] Prevent horizontal expansion of `TextArea` with `autoExpand={true}`.
```

Author: Changed it.
